policy_evaluation.go

```diff
@@ -0,0 +1,253 @@
+	Status           PolicyEvaluationStatus           `jsonapi:"attr,status"`
+	PolicyKind       PolicyKind                       `jsonapi:"attr,policy-kind"`
+	StatusTimestamps PolicyEvaluationStatusTimestamps `jsonapi:"attr,status-timestamps"`
+	ResultCount      *PolicyResultCount               `jsonapi:"attr,result-count"`
+	CreatedAt        time.Time                        `jsonapi:"attr,created-at,iso8601"`
+	UpdatedAt        time.Time                        `jsonapi:"attr,updated-at,iso8601"`
+
+	// The task stage this evaluation belongs to
+	TaskStage *PolicyAttachable `jsonapi:"relation,policy-attachable"`
+}
+
+// PolicyEvalutations describes all the policy evaluation related methods that the
+// Terraform Enterprise API supports.
+//
+// TFE API docs:
+// https://www.terraform.io/docs/cloud/api/policy-checks.html
+type PolicyEvaluations interface {
+	// **Note: This method is still in BETA and subject to change.**
+	// List all policy evaluations in the task stage. Only available for OPA policies.
+	List(ctx context.Context, taskStageID string, options *PolicyEvaluationListOptions) (*PolicyEvaluationList, error)
+}
+
+// policyEvaluation implements PolicyEvaluations.
+type policyEvaluation struct {
+	client *Client
+}
+
+// PolicyEvaluationListOptions represents the options for listing policy evaluations.
+type PolicyEvaluationListOptions struct {
+	ListOptions
+}
+
+// PolicyEvaluationList represents a list of policy evaluation.
+type PolicyEvaluationList struct {
+	*Pagination
+	Items []*PolicyEvaluation
+}
+
+// List all policy evaluations in a task stage.
+func (s *policyEvaluation) List(ctx context.Context, taskStageID string, options *PolicyEvaluationListOptions) (*PolicyEvaluationList, error) {
+	if !validStringID(&taskStageID) {
+		return nil, ErrInvalidTaskStageID
+	}
+
+	u := fmt.Sprintf("task-stages/%s/policy-evaluations", url.QueryEscape(taskStageID))
+	req, err := s.client.NewRequest("GET", u, options)
+	if err != nil {
+		return nil, err
+	}
+
+	pcl := &PolicyEvaluationList{}
+	err = req.Do(ctx, pcl)
+	if err != nil {
+		return nil, err
+	}
+
+	return pcl, nil
+}
+
+// Compile-time proof of interface implementation.
+var _ PolicySetOutcomes = (*policySetOutcome)(nil)
+
+// PolicySetOutcomes describes all the policy set outcome related methods that the
+// Terraform Enterprise API supports.
+//
+// TFE API docs:
+// https://www.terraform.io/docs/cloud/api/policy-checks.html
+type PolicySetOutcomes interface {
+	// **Note: This method is still in BETA and subject to change.**
+	// List all policy set outcomes in the policy evaluation. Only available for OPA policies.
+	List(ctx context.Context, policyEvaluationID string, options *PolicySetOutcomeListOptions) (*PolicySetOutcomeList, error)
+
+	// **Note: This method is still in BETA and subject to change.**
+	// Read a policy set outcome by its ID. Only available for OPA policies.
+	Read(ctx context.Context, policy_set_outcome_id string) (*PolicySetOutcome, error)
```

Comment:
nit: This probably shouldn't be snake_case
```suggestion
	Read(ctx context.Context, policySetOutcomeID string) (*PolicySetOutcome, error)
```